tests/data/doublestar_no_spaces.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3.7
+
+
+def function(**kwargs):
+    t = a**2 + b**3
```

Comment:
Should also test what happens if there *is* a space in the input.

Reply:
Maybe is it more a naming convention for the new option, maybe `skip` the chosen name is not the best, and I can change it. Regarding to the spaces question, it will yield the same output (meaning, `DOUBLESTAR` without spaces).

Does your comment mean, update the code and not merge yet, or does it mean: These changes are not convenient, forget about them?

I am sorry if I don't understand correctly it is my first PR on an open-source repo.

Reply:
Not sure, but I think Jelle meant you should add test cases which have spaces around the operators, which I think you've done in the code just below! @JelleZijlstra what do you reckon?

I'm not sure about the test writing practices of Black, but I see lots of tests being written a bit more condensed. In your file it's nice to see immediately what's expected from the function name, but another option would be to simply have all the cases in one succession. That's up for debate though!